modules/osd-monitoring-default-monitoring-targets.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
```

Comment:
Is my understanding correct that this module is not included in any of the assemblies? If that is the case, you might not need to add it.

Reply:
Yes, I deleted this topic from the assembly. I'm not going to include this module in the docs.